snapshot/sanitized/process_snapshot_sanitized_test.cc

```diff
@@ -98,7 +98,7 @@ void ChildTestFunction() {
   static StringAnnotation<32> non_allowed_annotation(kNonAllowedAnnotationName);
   non_allowed_annotation.Set(kNonAllowedAnnotationValue);
 
-  char string_data[strlen(kSensitiveStackData) + 1];
+  char* string_data = new char[strlen(kSensitiveStackData) + 1];
```

Comment:
While we don't care about this test, let's not introduce memory leaks into the code. Maybe use `std::string` or `std::vector<char>` here, to avoid leaking.